src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx

```diff
@@ -76,129 +77,181 @@ const Content = ({
                     )}
-                  </UsersColumn>
-                ) : (
-                  <SessionsColumn>
+                  </DailyColumn>
+
+                  <CrashesColumn>
                     {showPlaceholders ? (
-                      <StyledPlaceholder width="60px" />
-                    ) : defined(crashFreeSessions) ? (
-                      <CrashFree percent={crashFreeSessions} />
+                      <StyledPlaceholder width="30px" />
+                    ) : hasHealthData && defined(sessionsCrashed) ? (
+                      <Count value={sessionsCrashed} />
                     ) : (
                       <NotAvailable />
                     )}
-                  </SessionsColumn>
-                )}
-
-                <DailyColumn>
-                  {showPlaceholders ? (
-                    <StyledPlaceholder />
-                  ) : hasHealthData && defined(stats) ? (
-                    <ChartWrapper>
-                      <HealthStatsChart
-                        data={stats}
-                        height={20}
-                        period={activeStatsPeriod}
-                        activeDisplay={activeDisplay}
-                      />
-                    </ChartWrapper>
-                  ) : (
-                    <NotAvailable />
-                  )}
-                </DailyColumn>
-
-                <CrashesColumn>
-                  {showPlaceholders ? (
-                    <StyledPlaceholder width="30px" />
-                  ) : hasHealthData && defined(sessionsCrashed) ? (
-                    <Count value={sessionsCrashed} />
-                  ) : (
-                    <NotAvailable />
-                  )}
-                </CrashesColumn>
-
-                <IssuesColumn>
-                  <Tooltip title={t('Open in Issues')}>
-                    <Link
-                      to={getReleaseNewIssuesUrl(orgSlug, project.id, releaseVersion)}
-                    >
-                      <Count value={newGroups || 0} />
-                    </Link>
-                  </Tooltip>
-                </IssuesColumn>
-              </Layout>
-            </ProjectRow>
-          );
-        })}
-      </ClippedHealthRows>
+                  </CrashesColumn>
+
+                  <IssuesColumn>
+                    <Tooltip title={t('Open in Issues')}>
+                      <Link
+                        to={getReleaseNewIssuesUrl(orgSlug, project.id, releaseVersion)}
+                      >
+                        <Count value={newGroups || 0} />
+                      </Link>
+                    </Tooltip>
+                  </IssuesColumn>
+                </Layout>
+              </ProjectRow>
+            );
+          })}
+        </Collapsible>
+      </ProjectRows>
     </React.Fragment>
   );
 };
 
 export default Content;
 
+const ProjectRows = styled('div')`
+  position: relative;
+`;
+
+const ExpandButtonWrapper = styled('div')`
+  position: absolute;
+  left: 0;
```

Comment:
I'm aware that is from `clippedHealthRows.tsx` but if you want to clean things up, I think you can delete either `left` or `right`.

Reply:
I deleted both of them and set the width to 100% 👍 (deleting just one/both was not enough)